Doc/library/contextlib.rst

```diff
@@ -351,6 +351,43 @@ Functions and classes provided:
    .. versionadded:: 3.2
 
 
+.. class:: AsyncContextManager
+    
+   Similar as ContextManger only for async
```

Comment:
Actually, can you just add an example to https://docs.python.org/3/library/contextlib.html#contextlib.asynccontextmanager. I don't think we need to document the class itself.

Reply:
Maybe add a reference for more consistency. Something like `asynccontextmanager() uses AsyncContextDecorator so the context managers it creates can be used as decorators same as contextmanager.` And add ref from `AsyncContextDecorator` to `class:: AsyncContextManager`

Reply:
I wouldn't bother with anything but a 3 line example and saying it's possible starting with 3.10.